charts/workload-variant-autoscaler/templates/hpa.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.wva.hpa.enabled }}
+{{- if .Values.hpa.enabled }}
```

Comment:
Why was this chnaged? This adds regression at the top level. Helm treats the missing value as empty/false, so the template for hpa was skipped and e2es on openshift are failing